examples/keras/image_recognition/resnet50_fashion/quantization/ptq/README.md

```diff
@@ -0,0 +1,60 @@
+Step-by-Step
+============
+
+This document is used to list steps of reproducing TensorFlow keras model resnet50_fashion quantization and benchmark using Intel® Neural Compressor.
+This example can run on Intel CPUs and GPUs.
+
+
+# Prerequisite
+
+## 1. Environment
+
+### Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+
+### Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
```

Comment:
Use official tensorflow, not intel tensorflow. Let's add all the requirements into `requirements.txt`, like this:  
![image](https://user-images.githubusercontent.com/51692656/227100139-7bbdfd84-ce1a-4d47-bccf-e521c835187d.png)
![image](https://user-images.githubusercontent.com/51692656/227100208-1e003ffc-2a13-4d83-8554-a9dfe317ea7c.png)

Reply:
This is required by Neural Studio, so let's unified the style.